WordPress/troubleshooting-guides/tsg_existing_wordpress_installation_detected.md

```diff
@@ -0,0 +1,86 @@
+# Troubleshooting Guide for Existing WordPress Site Detected Warning
+
+## Overview
+If you see the following warning message on your WordPress site homepage:
+
+<img src="../media/existing-site-detected.png" width="800">
+
+This guide will explain the purpose of this warning and provide steps to resolve it yourself, with important notes on potential caveats.
+
+## Why Am I Seeing This Warning?
+This warning serves as a safeguard against potential data loss, especially in cases where the same MySQL database server is mistakenly linked to multiple WordPress sites or when an incorrect backup restore could erase existing data. This warning allows you to diagnose the situation if you believe it’s a false alarm or if you’re prepared to proceed with the assumption of a fresh installation.
+
+## Detailed Explanation
+The WordPress LAMP stack was originally designed for single-machine hosting, but with cloud-native, containerized hosting, WordPress must support independent scaling and fault tolerance. To achieve this, WordPress directories are hosted on a remote file system. Occasionally, delays in retrieving the current state from the remote file system can cause WordPress to interpret it as a fresh installation, potentially leading to data loss if this warning mechanism is not in place. Typically, a site restart resolves this issue if it’s due to a temporary synchronization delay.
+
+In other cases, the warning may persist due to:
+- Existing MySQL tables from a different WordPress setup
+- Partial WordPress installation that was not completed due to transient issues
+
+This is a warning, not an error, and you can proceed with resolving it if you’re familiar with your site setup.
+
+## Troubleshooting Steps
+
+1. **Restart the Site**  
+   Try restarting your site and wait a few minutes to see if the issue is transient. If the warning persists after 2-3 restarts, proceed with the steps below.
+
+2. **Access the SCM Site**  
```

Comment:
1. Restart the site and see if the issue goes away. (If this is caused by temporary I/O or storage issue at platform level, this would help).
2. Ensure that WEBSITES_ENABLE_APP_SERVICE_STORAGE is set to true.
3. Actually, Steps (2) to (8) could have been redundant as we removed that excessive check in the image. However, it is still good to keep them because it covers the scenario where /home/wp-locks/wp_deployment_status.txt file might have been accidentally deleted by the user.  
4. If the none of the above works, then there is definitely file data loss due to some underlying platform/storage layer issue. We should ask them to create support ticket to App Service / Storage Team so they can restore the file snapshot.